app/services/mailchimp_subscription.rb

```diff
@@ -0,0 +1,48 @@
+class MailchimpSubscription
```

Comment:
Can you add a unit test for this class please?

Reply:
Yes indeed! I'm in the process of adding the unit test now. My thought is if we stub out the service in the registrations controller test,  then the unit test should *actually* hit the Mailchimp API no? That way unit testing `MailchimpSubscription.create` tests API interaction works and stubbing out that method later would be fine because it's already tested. What are your thoughts?

Reply:
No worries @KevinMulhern. This makes sense and I've updated my pull request with the changes you've suggested. The test suite now runs with or without the environment variables related to the Mailchimp service. I was able to run the tests in question conditionally with the RSpec feature described [here](https://relishapp.com/rspec/rspec-core/docs/filtering/conditional-filters#the-:if-and-:unless-exclusions-stay-in-effect-when-there-are-explicit-inclusions). Please let me know if this is an acceptable alternative.

Reply:
Nice 👍 I haven't seen that feature of RSpec used before